src/stores/search.ts

```diff
@@ -0,0 +1,339 @@
+import { defineStore } from 'pinia'
+import {
+  computed,
+  ComputedRef,
+  reactive,
+  toRefs,
+  watch,
+} from '@nuxtjs/composition-api'
+// @ts-ignore
```

Comment:
nit: Can you silence this ESLint warning and add a Disable reason to note that we're removing this dependency soon and shouldn't worry about the missing types for now? Otherwise the warning will show on every subsequent PR. I opened an issue for us to discuss the warnings #1160